### DISCUSSION TITLE
GitHub Action

### DISCUSSION THREAD

**Discussion by hazcod** [aquasecurity/chain-bench#26] [Ideas]
A GitHub Action would be very nice to be able to run it.

**Comment by itaywol** [comment]
Working on it 😎

**Comment by itaywol** [comment]
After some 😷 corona time, and small discussion with @morwn decided to wait with the action a little bit, we want to print the output in json to stdout so you can pipe the result to `rego/jq`. I will update soon :)

**Comment by itaywol** [comment]
The first version of it is ready [here](https://github.com/itaywol/chain-bench-action).

We will move the repository into Aqua organization and release the action to the marketplace

**Comment by itaywol** [comment]
The Action is in the marketplace 😎 [here](https://github.com/marketplace/actions/chain-bench-action)
